parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -196,12 +196,24 @@ configureCNIIPTables() {
     fi
 }
 
-{{if NeedsContainerd}}
```

Comment:
can we still use this to save the ensureContainerD() method for docker nodes?

cloud-init data has size limit so we try our best to reduce the cse script size

Reply:
yep